WindowsServerDocs/networking/core-network-guide/cncg/server-certs/Refresh-Group-Policy.md

```diff
@@ -21,9 +21,7 @@ Membership in **Administrators**, or equivalent, is the minimum required to comp
 
 ### To refresh Group Policy on the local computer
 
-1.  On the computer where NPS is installed, open Windows PowerShell&reg; by using the icon on the taskbar.
+1.  On the computer where [**Network Policy Server NPS**](https://docs.microsoft.com/windows-server/networking/technologies/nps/nps-top) is installed, open Windows PowerShell&reg; by using the icon on the taskbar.
```

Comment:
Suggestions for the description text codestyle (use the page H1 title) and a relative link (works both on GitHub and docs.microsoft.com):
```suggestion
1.  On the computer where [**Network Policy Server (NPS)**](../../../technologies/nps/nps-top) is installed, open Windows PowerShell&reg; by using the icon on the taskbar.
```
You can test the link suggestion by visiting both pages in both sites.
1. docs.microsoft.com:
   - https://docs.microsoft.com/windows-server/networking/core-network-guide/cncg/server-certs/refresh-group-policy
   - https://docs.microsoft.com/windows-server/networking/technologies/nps/nps-top
2. github.com:
   - https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/networking/core-network-guide/cncg/server-certs/Refresh-Group-Policy.md
   - https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/networking/technologies/nps/nps-top.md

Reply:
@illfated.  sorry replaced link  ../../../     i dont think this work, so sorry at present I can not accept these commit changes

Reply:
Fair enough, I understand your concerns, especially since the Windows IT-Pro MS Docs team decided not to accept it right away (which is unfortunate, really). I can only assume they don't see a need for GitHub pages to be interlinked, as long as the linking works on docs.microsoft.com — but I digress.

Anyway, you might be able to accept a suggestion to adapt the description text alone, to make it match the page title:
```suggestion
1.  On the computer where [**Network Policy Server (NPS)**](https://docs.microsoft.com/windows-server/networking/technologies/nps/nps-top) is installed, open Windows PowerShell&reg; by using the icon on the taskbar.
```